api/v1alpha1/httproute_types.go

```diff
@@ -115,15 +116,21 @@ type HTTPRouteMatch struct {
 	// Default: "/"
 	Path *string `json:"path" protobuf:"bytes,2,opt,name=path"`
 
-	// HeaderType defines the semantics of the `Header` matcher.
+	// HeaderMatchType defines the semantics of the `Header` matcher.
+	//
+	// Support: core (Exact)
```

Comment:
I think @jpeach had mentioned the possibility of adding a `Contains` option here. That does seem like it could be useful, but I'm not sure how universally supported it would be.

Reply:
I think that it's fine to table that discussion until we have the general structure .

Reply:
Sorry for not responding to that comment earlier.
I opened up https://github.com/kubernetes-sigs/service-apis/issues/190 to track this.